.circleci/config.yml

```
@@ -31,10 +276,34 @@ workflows:
   version: 2
   test_and_deploy:
     jobs:
-      - test:
-          context:
-            - DockerHub
+      - test
+      - integration_mac_pass
```

You can use a [matrix](https://circleci.com/docs/2.0/configuration-reference/#matrix-requires-version-21) for this:
```suggestion
      - integration_pass:
          matrix:
            parameters:
              os: [linux, macos, windows]
```

And then you'd define a parametrized `integration_pass` job (not sure this is valid YAML, I'm just typing this out on GitHub)
```yaml
jobs:
  - integration_pass:
    parameters:
      os:
        type: executor
     steps:
       - checkout
       - ./$RAINFOREST_CLI -v --skip-update # set the env var to rainforest-cli or rainforest-cli.exe at the executor level
       - run:
           name: Create passing Rainforest run
           command: ./$RAINFOREST_CLI --skip-update run --run-group 9503
```